.github/workflows/make_release.yml

```diff
@@ -130,6 +130,7 @@ jobs:
         env:
           CLOUDSMITH_USER: ${{ secrets.CLOUDSMITH_USER }}
           CLOUDSMITH_API_KEY: ${{ secrets.CLOUDSMITH_API_KEY }}
+          TEST: "" # Build archives
```

Comment:
I think, it's not intuitive to see `TEST: ""` in the configuration.

I would suggest to make `TEST` optional instead. E.g., replace `$TEST` with `${TEST:-}`  in the `create_istio_release.sh`.